### PR TITLE
Fix docs typos in DiscordRestClient

### PR DESCRIPTION
**Pull request by InFTord**
# Summary
fixing a bunch of typos in [DiscordRestClient](https://github.com/DSharpPlus/DSharpPlus/blob/master/DSharpPlus.Rest/DiscordRestClient.cs)
# Details
fixed a bunch of inconsistent capitalization of abbreviations

# Changes proposed
* for example: in one part of docs, id was id, in second, it was Id, in third one ID.
# Notes
help

### CODE REVIEW

**Comment by InFTord** [comment]
im your local typo fixer x3

**Comment by OoLunar** [comment]
Please add subsequent typos in a singular PR, thank you.

**Comment by InFTord** [comment]
question: should i change stuff like words being not caps sometimes? (like id in one part of docs, Id in second one, and ID in third one)

**Comment by OoLunar** [comment]
If I say yes, how large do you plan on making this PR

**Comment by InFTord** [comment]
Good question, it prob will one file (DiscordRestClient) for now

**Comment by OoLunar** [comment]
Sounds good

**Comment by InFTord** [comment]
imma  gonna recheck it next morning, cuz im really tired
but i finished the file!

**Comment by InFTord** [comment]
as i said, editing DiscordRestClient with github editor is bad thing to do 😿

File: DSharpPlus.Rest/DiscordRestClient.cs [hunk 1]
```diff
@@ -823,8 +823,8 @@ public Task DeleteChannelPermissionAsync(ulong channel_id, ulong overwrite_id, s
         /// <summary>
         /// Edits channel overwrite
         /// </summary>
-        /// <param name="channel_id">Channel id</param>
-        /// <param name="overwrite_id">Overwrite id</param>
+        /// <param name="channel_id">Channel ID</param>
+        /// <param name="overwrite_id">OverwriteIDid</param>
```
Comment (ExaInsanity) [inline DSharpPlus.Rest/DiscordRestClient.cs:827]: this doesn't seem right

**Review by ExaInsanity — CHANGES_REQUESTED** [review]
I had no idea the xmldocs were this inconsistent with capitalization of abbreviations. Regardless:

File: DSharpPlus.Rest/DiscordRestClient.cs (hunk 1 above)
Comment (OoLunar) [reply DSharpPlus.Rest/DiscordRestClient.cs:827]: Failed sed moment

Comment (InFTord) [reply DSharpPlus.Rest/DiscordRestClient.cs:827]: ok i failed, gonna fix it rn

**Comment by InFTord** [comment]
i forgot, that i also found some typos

**Comment by InFTord** [comment]
found some typos yesterday (and forgot about them)

**Comment by InFTord** [comment]
anyways, on my way to create a new PR with basically same thing, but different files (it prob gonna be bigger)

**Review by ExaInsanity — APPROVED** [review]

**Comment by InFTord** [comment]
i regret adding lunar to review this PR :trollface:

**Comment by OoLunar** [comment]
@InFTord, since this PR is just documentation with no code changes, I encourage you to add to this PR instead of creating new PR's per file. Thank you for understanding.

**Review by OoLunar — APPROVED** [review]

**Comment by InFTord** [comment]
undocumented stuff?

**Pull request merged**